pkg/k8s/volumes/crud.go

```diff
@@ -37,6 +39,31 @@ func Create(ctx context.Context, dev *model.Dev, c *kubernetes.Clientset) error
 	return nil
 }
 
+func checkPVCValues(pvc *apiv1.PersistentVolumeClaim, dev *model.Dev) error {
```

Comment:
can we also take care of #624 with this change?

Reply:
Let's keep this PR short and fix #624 on a different PR